docs/advanced/additional-requests.rst

```diff
@@ -919,3 +889,118 @@ an :class:`~.HttpClient` instance:
 From the code sample above, we can see that every time an :class:`~.HttpClient`
 is created for Page Objects needing an ``http_client``, the specific **request
 implementation** from a given framework is injected to it.
+
+Downloader Behavior
+-------------------
+
+The Downloader should be able to accept an instance of :class:`~.HttpRequest`
+as the input and return an instance of :class:`~.HttpResponse`. This is important
+in order to handle and represent generic HTTP operations. The only time that
+it won't be returning :class:`~.HttpResponse` would be when it's raising exceptions
+(see :ref:`framework-exception-handling`).
+
+The Downloader should be able to properly resolve **redirections** except when
+the method is ``HEAD``. This means that the :class:`~.HttpResponse` that it'll
```

Comment:
> except when the method is ``HEAD``

That's an interesting behavior. It seems right to me. I wonder how easy would it be to implement it though. For example, Scrapy seems to redirect in case of HEAD: https://github.com/scrapy/scrapy/blob/b2afcbfe2bf090827540d072866bef0d1ab3a3e8/scrapy/downloadermiddlewares/redirect.py#L102

Reply:
Thanks for raising this @kmike ! I didn't that this was the default behavior of Scrapy. 

I think the easiest route would be for **scrapy-poet** to set `meta={'dont_redirect'=True}` when it sees a `HEAD` request method. Added a note in the TODO section of https://github.com/scrapinghub/scrapy-poet/pull/62 so we wouldn't forget.